examples/custom_client.rs

```diff
@@ -26,7 +26,7 @@ where
     let request = http::Request::get("https://example.com")
         .header(
             http::header::AUTHORIZATION,
-            format!("Bearer {}", token.as_str()),
+            format!("Bearer {}", token.access_token().ok_or("no access token")?),
```

Comment:
did I miss something, or should this be `token.token()`? According to your changes in the `types` module.